linalg/densemat.hpp

```diff
@@ -758,6 +758,30 @@ class DenseTensor
 
    long MemoryUsage() const { return nk*Mk.MemoryUsage(); }
 
+   /// Shortcut for mfem::Read( GetMemory(), Size(), on_dev).
```

Comment:
To be precise, `Size()` should be replaced by `TotalSize()` in the comments.

Reply:
Fixed 👍